source/app/post_init.py

```diff
@@ -121,20 +121,18 @@ def run_post_init(development=False):
     log.info(f'IRIS {app.config.get("IRIS_VERSION")}')
     log.info("Running post initiation steps")
 
-    conn = None
-
     if os.getenv("IRIS_WORKER") is None:
         # Attempt to connect to the database with retries
         log.info("Attempting to connect to the database...")
         for i in range(retry_count):
             log.info("Connecting to database, attempt " + str(i+1) + "/" + str(retry_count))
             conn = connect_to_database(db_host,db_port)
-            if conn is not None:
+            if conn:
```

Comment:
Hi! Thanks for the PR! 

Any reasons why the `is not None` is removed? 
thanks!

Reply:
Hi!

First of all, happy new year!

If you look closer at the connect_to_database function, it is returning True or False, not something or None.

So, you're not checking the return value (conn) properly with "if conn is not None" ; but with simply "if conn" it is now checking properly the correct return values.

Hope this is clear enough.

Thanks for the review!